transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -140,13 +140,17 @@ static void netty_epoll_native_eventFdWrite(JNIEnv* env, jclass clazz, jint fd,
     }
 }
 
-static void netty_epoll_native_eventFdRead(JNIEnv* env, jclass clazz, jint fd) {
-    uint64_t eventfd_t;
+static jlong netty_epoll_native_eventFdRead(JNIEnv* env, jclass clazz, jint fd) {
+    uint64_t val;
 
-    if (eventfd_read(fd, &eventfd_t) != 0) {
+    if (eventfd_read(fd, &val) != 0) {
+        if (errno == EAGAIN) {
+            return (jlong) 0;
```

Comment:
I wonder if we should better return a negative value ?

Reply:
Imho `0` is better here, `EAGAIN` does mean exactly that a value `0` of was read. From [eventfd manpage](http://man7.org/linux/man-pages/man2/eventfd.2.html):
> If the eventfd counter is zero at the time of the call to read(2), then the call either blocks until the counter becomes nonzero (at which time, the read(2) proceeds as described above) or fails with the error `EAGAIN` if the file descriptor has been made nonblocking.

Reply:
I will add a comment for that too :)

Reply:
make sense... thanks